src/main/resources/i18n/messages_cs.properties

```diff
@@ -21,6 +21,7 @@ label.client.cypherdetail.solution=Řešení
 label.client.cypherdetail.verify=Ověřit
 label.client.cypherdetail.finalplacedetail=Detail vyhlášení výsledků
 label.client.cypherdetail.skipped=Tuto šifru jste přeskočili
+label.client.cypherdetail.gamenotready=Hru ještě nění možné zahájit. Počkejte na pokyny pořadatele
```

Comment:
Ještě bych dal tečku nakonec a je to dobrý :+1:

Reply:
Když už budu přidávat tečku, klidně změním počkejte na vyčkejte :-)